packages/shared/components/KeyListRow.svelte

```diff
@@ -1,30 +1,43 @@
 <script lang="ts">
     import { Pill } from './pills'
-    import { Text, Modal, MeatballMenuButton, MenuItem } from 'shared/components'
+    import { Text, Modal, MeatballMenuButton, MenuItem, TooltipIcon } from 'shared/components'
     import { localize } from '@core/i18n'
+    import { truncateString } from 'shared/lib/core/utils'
+    import { selectedWalletId, selectedWallet } from 'shared/lib/core/wallet'
+    import { updateActiveWalletPersistedData } from 'shared/lib/core/profile'
 
     export let key: string
 
     let modal: Modal | undefined
+    const truncatedKey: string = truncateString(key, 12, 12)
```

Comment:
Since we have truncated the key I think it is no longer necessary for the size of the popup to be `large`